app/helpers/patternfly_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module PatternflyHelper
+  def pf_button_in_title(label, href)
+    link_to label, href, class: %w[pf-c-button pf-m-primary pf-c-button__in-title]
```

Comment:
We have some pf button helpers [here](https://github.com/3scale/porta/blob/master/app/helpers/buttons_helper.rb#L147)
Maybe you can add it in same place

Reply:
Yes, I saw them and my idea was to move them inside this new helper. Otherwise it's a bit of a pain to maintain.. if we're to move everything to PF, better start cleaning up the older helpers.